### DISCUSSION TITLE
Topology View Images set is inconsistent with existent objects

### DISCUSSION THREAD

**Discussion by andrei-korshikov** [mattieserver/netbox-topology-views#404] [General]
Fresh installation of NetBox v3.6.3, plugin version 3.8.0. The only role I have is "Campus" (and only two devices, but that doesn't matter). I understand why Images page hides (well, doesn't show) all that fancy .svg files - I don't have objects with correspondent objects/roles. I personally would prefer to see them all, but that's a matter of taste.

But, if plugin filters images and shows only relevant (existent in the topology), why do I see "Circuit", "Power Feed" and "Power Panel"? And, if "Power Feed" is needed to be always visible, why is it blank?

**Screenshot:**
![image](https://github.com/mattieserver/netbox-topology-views/assets/15959858/d5416eeb-dcf0-4851-9848-0948c2cc8076)

PS: Dear author, thank you for really amazing plugin. It pleases the eye.

**Comment by mattieserver** [comment]
Hi,

"Circuit", "Power Feed" and "Power Panel" are special cases. Since they are not seen as a device in netbox we cannot get the device role to assign an image to.
But you are right, the power feed should have an actual icon. I will look into it.
